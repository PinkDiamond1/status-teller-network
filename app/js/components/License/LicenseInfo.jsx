import React, {Component} from 'react';
import {Row, Col, Card, CardBody} from 'reactstrap';
import PropTypes from 'prop-types';
import {withNamespaces} from 'react-i18next';

class LicenseInfo extends Component {
  render() {
    const t = this.props.t;
    return (
      <div className="mt-5 mb-5">
        <Row>
          <Col xs={12}>
            <h2 className="text-center">{t('sellerLicenseInfo.title')}</h2>
          </Col>
        </Row>

        <Card color="primary" className="mt-5 py-4">
          <CardBody className="text-center">
            {t('sellerLicenseInfo.fee')}
          </CardBody>
        </Card>

        <Card color="primary" className="mt-5 py-4">
          <CardBody className="text-center">
            {t('sellerLicenseInfo.stake')}
          </CardBody>
        </Card>
      </div>
    );
  }
}

LicenseInfo.propTypes = {
  t: PropTypes.func
};

export default withNamespaces()(LicenseInfo);
