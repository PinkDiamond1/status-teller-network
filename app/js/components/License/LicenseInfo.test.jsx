import React from "react";
import { shallow } from "enzyme";

import LicenseInfo from "./LicenseInfo";

describe('LicenseInfo', () => {
  it('should render correctly', () => {
    const component = shallow(<LicenseInfo/>);
  
    expect(component).toMatchSnapshot();
  });
});
