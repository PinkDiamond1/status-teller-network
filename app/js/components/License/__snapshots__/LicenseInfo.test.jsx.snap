// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LicenseInfo should render correctly 1`] = `
<div
  className="mt-2"
>
  <Row
    tag="div"
  >
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
      xs={12}
    >
      <h2
        className="text-center"
      />
    </Col>
  </Row>
  <div
    className="fee-card mt-3 py-3 text-center rounded font-weight-bold"
  >
    <br />
  </div>
  <div
    className="stake-card mt-3 py-3 text-center rounded font-weight-bold"
  >
    <br />
  </div>
</div>
`;
